apps/omg_watcher/lib/omg_watcher/api/status.ex

```diff
@@ -36,6 +38,7 @@ defmodule OMG.Watcher.API.Status do
   last mined child block number and it's timestamp, and a flag indicating if watcher is syncing with Ethereum.
   """
   @spec get_status() :: {:ok, t()} | {:error, atom}
+  @decorate measure_event()
```

Comment:
as this function body is quite big ... would the result of the measurement tell you anything meaningful? would the result tell you where in the body it spent a lot of time?

Reply:
yes, it tells me how much time spend in RootChain, and ExitProcessor and BlockGetter
(in this modules is also measure_event)